docs/source/tutorials/configure_systems.rst

```diff
@@ -7,6 +7,19 @@ A system is the environment (OS, hardware spec, device platform, supported EP) t
 is evaluated on. It can thus be the **host** of a Pass or the **target** of an evaluation. This document describes
 how to configure the different types of Systems.
 
+Accelerator Configuration
+-------------------------
+For each **host** or **target**, it could specify multiple accelerators associated with it. Each accelerator could have the following attributes.
+
+* :code:`device`: The device type of the accelerator. It could be "cpu", "gpu", "npu", etc. Please refer to the API documentation for the full list of supported devices.
+* :code:`execution_providers`: The execution provider list that are supported by the accelerator. For e.g. ["CUDAExecutionProvider", "CPUExecutionProvider"].
+
+Note:
+
+- The accelerators for local system or python system is optional. If not provided, Olive will get the available execution providers installed in current local machine and infer its device.
+- For local system or python system, either device or execution_providers is optional but not both if the accelerators are specified. If device or execution_providers is not provided, Olive will infer the device or execution_providers if possible.
+- For docker system and AzureML system, both device and execution_providers are mandatory. Otherwise, Olive will raise an error.
+
```

Comment:
Can you add some examples of different ways of specifying the accelerator configs.
(1. both given, 2. ep given, device inferrered, 3. device given, ep inferred, 4. none given, inferred from local environment, etc)?

Reply:
As I describe offline, there will be a followup work item https://aiinfra.visualstudio.com/Model%20optimization%20Toolkit/_workitems/edit/22491/ for how to use system. I will update then